examples/sol-anchor-contract/scripts/invoke.ts

```diff
@@ -0,0 +1,95 @@
+const fs = require('fs');
```

Comment:
This in reality is a test file so maybe I would call it `anchor test` instead of `anchor invoke`